core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java

```diff
@@ -2557,12 +2555,6 @@ public TestMetadataManagerBuilder withCatalogManager(CatalogManager catalogManag
             return this;
         }
 
-        public TestMetadataManagerBuilder withFeaturesConfig(FeaturesConfig featuresConfig)
-        {
```

Comment:
cc @sopel39

Reply:
I think it's unused, but let @dain check it since he added it